cadence/contracts/interfaces/DeFiActions.cdc

```diff
@@ -656,6 +730,9 @@ access(all) contract DeFiActions {
             self._rebalanceSink = outSink
             self._rebalanceSource = inSource
             self._selfCap = nil
+            self._lastRebalanceTimestamp = getCurrentBlock().timestamp
+            self._recurringConfig = recurringConfig
+            self._scheduledTransactions <- {}
```

Comment:
Schedule transactions inner state could be dropped in favor of using the state provided by the manager contract. In fact, it stores too scheduled transactions by handler, managing cleanup and cancelation. This could remove a lot of logic around managing scheduled transactions here and make it lightweight https://github.com/onflow/flow-core-contracts/blob/aab7a9dda680bbac49b1d12f0f69b18e8490a943/contracts/FlowTransactionSchedulerUtils.cdc#L118

Reply:
Or maybe it is intended for internal checks and avoiding interfering external schedule?